frontends/php/screenconf.php

```diff
@@ -209,7 +209,7 @@
 
 		if (getRequest('form') === 'full_clone') {
 			$output = ['resourcetype', 'resourceid', 'width', 'height', 'x', 'y', 'colspan', 'rowspan', 'elements',
-				'valign', 'haligh', 'style', 'url', 'max_columns'
+				'valign', 'halign', 'style', 'url', 'max_columns'
```

Comment:
I'm not sure what this code was doing, but it wasn't doing the right thing for `halign` before...